sklearn/decomposition/_lda.py

```diff
@@ -109,6 +113,7 @@ def _update_doc_distribution(
             cnts = X[idx_d, ids]
 
         doc_topic_d = doc_topic_distr[idx_d, :]
+
```

Comment:
May we revert this unrelated change?

(It makes `git blame` easier to do)

Reply:
Got it. reverted.